html/pfappserver/lib/pfappserver/I18N/i_default.po

```diff
@@ -6996,44 +6996,44 @@ msgstr "pending"
 msgid "pf::pki_provider::packetfence_pki"
 msgstr "PacketFence PKI"
 
-# pf::filter (Portal Profile Filters)
-msgid "pf::profile::filter::connection_type"
+# pf::condition::profile (Portal Profile Filters)
+msgid "pf::condition::profile::connection_type"
```

Comment:
Being picky but I think their namespace shouldn't be linked to pf::condition as they are not there anymore.

Maybe profile::filter::connection_type ?